scss/components/_typeahead.scss

```diff
@@ -9,68 +9,76 @@ $DBtypeahead-namespace: "c-typeahead" !default;
 $DBtypeahead-background: color(white) !default;
 
 .#{$DBinput-namespace}--typeahead {
-  &.is-active {
-    border-bottom: 0;
-    border-bottom-left-radius: 0;
-    border-bottom-right-radius: 0;
-  }
+    &.is-active {
+        border-bottom: 0;
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+    }
 }
 
-/**
+/*!
  * Typeahead component
  */
 .#{$DBtypeahead-namespace} {
-  display: none;
-  position: absolute;
-  top: 100%;
-  left: 0;
-  right: 0;
-  margin-top: -1px;
-  overflow: scroll;
-  max-height: 180px;
-
-  line-height: 1.5;
-
-  background-color: $DBtypeahead-background;
-  background-clip: padding-box;
-  border-radius: 0 0 4px 4px;
-  border: 1px solid color(gray, x-dark, 0.1);
-  box-shadow: 0 1px 2px color(gray, x-dark, 0.1);
-
-  &.is-open {
-    display: block;
-  }
-
-  &__result {
+    display: none;
+    overflow: scroll;
+    position: absolute;
```

Comment:
Properties should be ordered display, overflow, overflow, position, top, right, left, left, max-height, margin-top, line-height, border, border-radius, border-radius, box-shadow, background-color, background-color